### DISCUSSION TITLE
Font size looks different in design interface and user interface

### DISCUSSION THREAD

**Discussion by su-aac** [UNC-Project-Open-AAC/OS-DPI#233] [Help]
When adjusting the font size, the design interface does not reflect the real size of the user interface. I have to go to the user interface side to see how it looks there to adjust the font size on the design interface side. I wonder if the design interface can be adjusted to mirror the user interface in a more intuitive way.

Font size in the design interface:
<img width="960" alt="image" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/123387291/bf5a4156-782a-44c1-9f18-386a6fe17b92">

Font size in the user interface:
<img width="960" alt="image" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/123387291/10405668-5ca7-401f-b796-da8c630b8238">

[magnify_feedback_12_12.23.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13729044/magnify_feedback_12_12.23.zip)

**Comment by gbishop** [comment]
You are using "px" units on your sizes. Those are absolute. You want to use "em" as the unit. An em is approximately equivalent to 16px when things are NOT scaled. The CSS I'm using automagically scales the fonts to the screen size by adjusting the em size. 

I suggest replacing your 25px with 2em and your 40px with 3em as a starting point, then adjust from there.
